documentation/man/features/mail.rst

```diff
@@ -29,3 +30,7 @@ OPTIONS
 
 -v, \--verify:
   Verify that all the settings needed are set and valid.
+
+-i, \--interactive:
+  This checks your current configuration and prompts about missing options and
+  changes.
```

Comment:
I think we should pass through all configurations, even if they are already set. Maybe we can ask the user if they want to replace or not in each option.